File: app/src/main/java/com/company/redcode/royalcryptoexchange/adapter/CustomSpinnerAdapter.java
package com.company.redcode.royalcryptoexchange.adapter;

import java.util.ArrayList;
import android.app.Activity;
import android.content.Context;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.BaseAdapter;
import android.widget.ImageView;
import android.widget.TextView;

import com.company.redcode.royalcryptoexchange.R;
import com.company.redcode.royalcryptoexchange.models.PaymentMethod;

public class CustomSpinnerAdapter extends BaseAdapter {

    Context c;
    ArrayList<PaymentMethod> objects;

    public CustomSpinnerAdapter(Context context, ArrayList<PaymentMethod> objects) {
        super();
        this.c = context;
        this.objects = objects;
    }

    @Override
    public int getCount() {
        return objects.size();
    }

    @Override
    public Object getItem(int position) {
        return objects.get(position);
    }

    @Override
    public long getItemId(int position) {
        return 0;
    }

    public View getView(int position, View convertView, ViewGroup parent) {

        PaymentMethod cur_obj = objects.get(position);
        LayoutInflater inflater = ((Activity) c).getLayoutInflater();
        View row = inflater.inflate(R.layout.single_row_payment_spinner, parent, false);
        TextView tv_bnk_name = row.findViewById(R.id.tv_bnk_name);
        TextView tv_bnk_title = row.findViewById(R.id.tv_bnk_title);
        tv_bnk_name.setText(cur_obj.getBankName());
        tv_bnk_title.setText(cur_obj.getType());
        return row;
    }
}

File: app/src/main/java/com/company/redcode/royalcryptoexchange/adapter/CustomSpinnerAdapter.kt
package com.company.redcode.royalcryptoexchange.adapter

import android.content.Context
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.ArrayAdapter
import com.company.redcode.royalcryptoexchange.models.PaymentMethod
import android.widget.TextView
import com.company.redcode.royalcryptoexchange.R


class CustomSpinnerAdapter( context: Context?, var resource: Int, var list: ArrayList<PaymentMethod>?) :
        ArrayAdapter<PaymentMethod>(context, resource, list) {

    override fun getView(position: Int, convertView: View?, parent: ViewGroup?): View {
        return createItemView(position, convertView!!, parent!!)
    }

    override fun getDropDownView(position: Int, convertView: View?, parent: ViewGroup?): View {
        return createItemView(position, convertView!!, parent!!)
    }

    private fun createItemView(position: Int, convertView: View, parent: ViewGroup): View {
        val view = LayoutInflater.from(context).inflate(resource, parent, false)

        val tv_bnk_name = view.findViewById(R.id.tv_bnk_name) as TextView
        val tv_bnk_title = view.findViewById(R.id.tv_bnk_title) as TextView
        tv_bnk_title.setText(list!![position].Account)
        tv_bnk_name.setText(list!![position].BankName)


        return view
    }
}